sdk/iOS/ZumoE2ETestApp/ZumoE2ETestApp/ZumoPushTests.m

```diff
@@ -105,31 +105,13 @@ - (BOOL)compareExpectedPayload:(NSDictionary *)expected withActual:(NSDictionary
     return allEqual;
 }
 
-- (NSDictionary *)zumoPayloadToApsPayload:(NSDictionary *)originalPayload {
-    NSMutableDictionary *result = [[NSMutableDictionary alloc] init];
-    NSMutableDictionary *aps = [[NSMutableDictionary alloc] init];
-    [result setValue:aps forKey:@"aps"];
-    id alert = originalPayload[@"alert"];
-    if (alert) {
-        [aps setValue:alert forKey:@"alert"];
-    }
-    
-    id badge = originalPayload[@"badge"];
-    if (badge) {
-        [aps setValue:badge forKey:@"badge"];
-    }
-    
-    id sound = originalPayload[@"sound"];
-    if (sound) {
-        [aps setValue:sound forKey:@"sound"];
-    }
+- (NSDictionary *)zumoPayloadToApsPayload:(NSDictionary *)payload {
```

Comment:
do we really need a function to do this?  The server is inconsistent in what it sends back?